### ISSUE TITLE
When ResourceRule class has configure method, rflint says the rule is unknown.

### ISSUE THREAD

**Issue by jaloren** [boakley/robotframework-lint#30]
I have a resource rule that needs to process a json configuration file. I pass this file in via the --configure option. I verified that the class is being picked up by the linter by  executing the rflint --list.

However, when I execute the rflint command, it says the rule is unknown. If I just change the abstract class that my concrete class is subclassed from to GeneralRule, then the rule is actually applied by rflint. 

Here's an example class.

```
class LibraryImport(ResourceRule):
    severity = ERROR

    def configure(self, jsonfile):
        self.jsonfn = jsonfile
        self.allowed_libs = json.load(open(jsonfile,'r'))


    def apply(self,resource):
        settings_table = [table for table in resource.tables if table.name == 'Settings']
        if not settings_table:
            return 
        else:
            settings_table = settings_table[0]
        for row in settings_table.rows:
            if re.search('library',row[0],re.IGNORECASE) and row[1] not in self.allowed_libs:
                msg = "A library '{0}' is being imported in a resource file but this library is not allowed to be used. Allowed libraries are: {1}".format(row[1],self.allowed_libs)
                self.report(resource,msg,row.linenumber)
```

**Comment by boakley** [comment]
Are you using the --rulefile option to load that file, or are you putting the module somewhere in your PYTHONPATH?

**Comment by boakley** [comment]
Never mind, it doesn't matter. I found the bug. Thank you for reporting it!

**Comment by boakley** [comment]
This will be available in the next release (0.6)
